CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 2024-03-16
+
+### Updated images
+
+- datalens-us: 0.168.0 -> 0.175.0
+- datalens-ui: 0.1410.0 -> 0.1440.0
+
+### Changes
+
+- Unify empty states styles ([ui:#731](https://github.com/datalens-tech/datalens-ui/pull/731))
```

Comment:
May be it's better to add about changing styles in markdown? Affects on text dash widget